config/galaxy.ini.sample

```diff
@@ -633,6 +637,31 @@ nglims_config_file = tool-data/nglims.yaml
 # https://FQDN/galaxy-prefix/gie_proxy for proxying
 #dynamic_proxy_prefix=gie_proxy
 
+# The Golang proxy has a large number of extra options which were not available
+# to the NodeJS proxy, as the Golang proxy takes over container management itself.
+# This should actually be a preferrable situation, as there will be less hackery
+# required in GIE configs.
+
+# The length of time a container isn't accessed, before it is culled.
+# Previously, individual containers set their own time limit before killing
```

Comment:
I think it's easier for everyone if they understand the config options, not how it used to be in the past ... but that's my personal opinion ;).